target/scripts/startup/setup-stack.sh

```diff
@@ -89,6 +89,25 @@ function _setup_amavis
   fi
 }
 
+function _setup_rspamd
+{
+  _log 'warn' 'Rspamd support is under active development, expect breaking changes at any time'
+
+  if [[ ${ENABLE_AMAVIS} -eq 1 ]] || [[ ${ENABLE_SPAMASSASSIN} -eq 1 ]]
+  then
+    _shutdown 'You cannot run Amavis/SpamAssassin and Rspamd at the same time'
+  fi
+
+  if [[ ${ENABLE_CLAMAV} -eq 0 ]]
+  then
+    _log 'debug' 'Rspamd will not use ClamAV (which is disabled)'
```

Comment:
```suggestion
    _log 'debug' 'Rspamd will not use ClamAV (which has not been enabled)'
```

I think the convention we have with other features (some dovecot ones?), is to have a different extension that is renamed with `mv`. In this case since ClamAV is disabled by default, it should probably be the other way around and if `-eq 1` then log about configuring Rspamd for ClamAV followed by the rename to enable the config?

Reply:
👍🏼